client/components/loading-spinner/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Spinner from 'components/spinner';
+
+const LoadingSpinner = ( { inline } ) => {
```

Comment:
Calypso also has its own loading button - [`<SpinnerButton />`](https://wpcalypso.wordpress.com/devdocs/design/spinner-button) - which we can use instead, but it doesn't seem to be working atm.